generate_smr_inputs/1.0/generate_smr_inputs.R

```diff
@@ -1,12 +1,13 @@
-#!/usr/bin/env Rscript
+# /usr/bin/env Rscript
 
 # Usage:
 #   Rscript generate_smr_inputs.R --genome <path/to/genome/master/seg> --capture <path/to/genome/master/seg> --output_path <path/to/output/folder>
 #            --all_sample_sets <path/to/sample_sets> --case_set <case_set>
 #
 #   Example: Rscript generate_smr_inputs.R --genome /projects/rmorin/projects/gambl-repos/gambl-sgillis/results/gambl/gistic2-1.0/00-inputs/genome--projection/all--grch37.seg --capture /projects/rmorin/projects/gambl-repos/gambl-sgillis/results/gambl/gistic2-1.0/00-inputs/capture--projection/all--grch37.seg
-#             --output_dir /projects/rmorin_scratch/sgillis_temp/lcr-scripts/generate_smr_inputs/1.0/ --all_sample_sets /projects/rmorin/projects/gambl-repos/gambl-sgillis/data/metadata/level3_samples_subsets.tsv --case_set FLs_with_LSARP_Trios
+#             --output_dir /projects/rmorin_scratch/sgillis_temp/lcr-scripts/generate_smr_inputs/1.0/ --all_sample_sets /projects/rmorin/projects/gambl-repos/gambl-sgillis/data/metadata/level3_subsetting_categories.tsv --case_set FLs_with_LSARP_Trios
 #
+# TO DO: update this description
```

Comment:
Just wonder if this is already completed? Because the description seems updated :smile:

Reply:
Fixed!